packages/schematics/angular/class/schema.json

```diff
@@ -3,7 +3,7 @@
   "$id": "SchematicsAngularClass",
   "title": "Angular Class Options Schema",
   "type": "object",
-  "description": "Creates a new generic class definition in the given or default project.",
+  "description": "Creates a new, generic class definition in the given or default project.",
```

Comment:
Just a minor fixup to make the description consistent with other schematics in this package, for example
https://github.com/angular/angular-cli/blob/07bf174759882f4616ed276835dd4cb81491d49c/packages/schematics/angular/e2e/schema.json#L7